README.md

```diff
@@ -412,12 +441,22 @@ axios at the same time.
 }
 ```
 
-All the other parameters will be passed directly to the [`axios.create`](https://github.com/mzabriskie/axios#creating-an-instance) method.
+### RedisStore(client, [, hashKey])
+
+RedisStore allow you to cache requests on server using [redis](https://redis.io/).
+Create a `RedisStore` instance. Takes `client` (`RedisClient`) and optional `hashKey` (name of hashSet to be used in redis).
+
+#### client
+
+```js
+    // Using redis client https://github.com/NodeRedis/node_redis
```

Comment:
Should we add `redis` client as a peer dependency?
Or maybe just add a comment about which version is supported in the readme?

Reply:
I'm with adding comment with the version supported till now.